src/components/Style.css

```diff
@@ -0,0 +1,151 @@
+.left_style {
+  font-size: 14;
+  margin-left: 0;
+  color: #F35B8C;
+  text-align: left; 
+}
+
+.right_style {
+  font-size: 14;
+  margin-right: 0;
+  color: #F35B8C;
+  text-align: right;
+}
+
+
+.reddiv{width:auto;height: 81vh; background-color:  #F35B8C;}
+
+.bluediv{width:auto;height: 88vh; background-color:#69B5FF;}
+
+.textdiv{
+  align-self: center;
+  margin-left: 5%;
+  margin-top: 15%;
+}
+
+.labelclass{
+  padding-right:0;
+  color: white;
+  margin-left: 0;
+  margin-right:100;
+}
+
+
+.split1 {
+  height: 250%;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  overflow-x: hidden;
+
+}
+
+/* Control the left side */
+.left1 {
+  left: 0;
+  width: 31.5%;
+  background-color: #F35B8C;
+}
+/* Control the center side */
+.center1 {
+  left: 31.5%;
+  width: 67%;
+  background-color: rgb(235, 228, 228);
+  
+  overflow: hidden;
+}
+/* Control the right side */
+.right1 {
+  right: 0;
+  width: 37%;
+  background-color: #ffffff;
+  height:100%;
+}
+
+.rowheader{
+  display:inline-flex;
+  height:20%;
+
+ 
+
+ 
+  /* width:200%; */
+}
+.settings{
+  /* float:left; */
+ 
+  background: #cac9e7;
+  margin-top: 0;
+  display: inline-block;
+  text-align: center;
+  padding-top: 10%;
+}
+.exit{
+  /* float:right; */
+  background: #464475;
+  margin-top: 0;
+  display: inline-block;
+  text-align: center;
+  padding-top: 10%;
+}
+
+.header {
+  display: inline-block;
+  margin-top: 0;
+  /* float:left; */
+  background: #A4A1FB;
+  color: white;
+  font-size: 37px;
+  text-align: center;
+  padding: 10%;
+  overflow: auto;
+  font-weight: bold;
+  font-family: Bold 35px/37px source sans pro;
+  }
+
+  
+  .income{
+    background-color: rgb(0, 94, 131);
+    text-align: left;
+    color: wheat;
+    width: 50%;
+  }
+  
+  .expense{
+    background-color:rgb(255, 0, 76);
+    text-align: left;
+    color: wheat;
+    width: 36%;
+  }
+  .chart-container {
+    width: "15%";
+    height: "10%";
+    margin-top: "10%";
+    padding: "2%";
+  }
+  .switch{
+    background-color: seashell;
+  }
+  
+  #cbarhart{
+    /* margin-top: px; */
+    width: "50%";
+    height: "10%";
+    /* padding: 5px; */
+  }
+  #cbar{
+    width: 50%;
+    padding-bottom: 45px;
+  }
+  #cline{
+    width: 80%;
+    padding-bottom: 45px;
+   
+  }
+  #barchart{
+    margin-top: 200px;
+    width: 350px;
+    height: 80px;
+    padding: 5px;
+  }
+  
```

Comment:
@Sruthyparvathy Follow hyphen delimited string naming convention for class name and id